CHANGELOG.md

```diff
@@ -16,6 +16,36 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The `merlin monitor` command will now keep an allocation up if the queues are empty and workers are still processing tasks
 - Add the restart keyword to the specification docs
 
+### Changed
```

Comment:
in general I'd recommend shrinking this down, maybe just the top 2 layers. some of the comments are informal notes and we should keep this a clean record